src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1096,6 +1098,18 @@ namespace winrt::TerminalApp::implementation
         return std::nullopt;
     }
 
+    // Method Description:
+    // - returns a com_ptr to the currently focused tab. This might return null,
+    //   so make sure to check the result!
+    winrt::com_ptr<Tab> TerminalPage::_GetFocusedTab()
+    {
+        if (auto index{ _GetFocusedTabIndex() })
+        {
+            return _GetStrongTabImpl(*index);
+        }
+        return nullptr;
```

Comment:
Honestly, I don't know. But we're using this pattern _all over_, and I thought this would be useful. I think there can be some weird async things, especially near teardown, where there's not a focused tab.